Makefile

```diff
@@ -31,7 +31,7 @@ ifneq ($(WERROR),0)
 endif
 
 ifeq ($(OPTIMIZATION_ON),0)
-  OPTFLAGS := -O0
+  OPTFLAGS := -Og
```

Comment:
Personally I have gotten better debugging results with `-O0` than with `-Og`, so I suggest just changing this back